packages/autorest.typescript/src/restLevelClient/transforms/transform.ts

```diff
@@ -60,6 +60,9 @@ export function transform(model: CodeModel): RLCModel {
     }
   };
   rlcModel.sampleGroups = transformRLCSampleData(model, rlcModel);
+  rlcModel.options!.generateSample =
```

Comment:
I didn't see tests/samples generated in todo_non_branding project.  Are we able to generate them also for non-branding projects?